### ISSUE TITLE
Incorrect column number for whitespace conventions.

### ISSUE THREAD

**Issue by ldec** [PyCQA/pylint#1649]
### Steps to reproduce

1. Create test.py
```python
var1 =  1
var2 = (  2, 3)
```

2. Run `pylint test.py`
```
C:  1, 0: Exactly one space required after assignment
var1 =  1
     ^ (bad-whitespace)
C:  2, 0: No space allowed after bracket
var2 = (  2, 3)
       ^ (bad-whitespace)
```

### Current behavior

The circumflex accent placed under the code shows the incriminated code correctly, however the column number is 0.

### Expected behavior

The column number shouldn't be 0 and should represent the column indicated by the circumflex accent.

### pylint --version output
```
No config file found, using default configuration
pylint 1.7.2, 
astroid 1.5.3
Python 3.5.3+ (heads/3.5:d51d093, Jun  6 2017, 15:15:51) 
[GCC 4.8.4]
```

**Comment by brycepg** [comment]
I'm almost done with this one. Thank you for the issue
